cmake/module/GenerateMPCode.cmake

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+macro(generate_from_capnp capnp_relpath)
```

Comment:
In commit "cmake: Add MULTIPROCESS option " (15ce0774f84281abff008a30890ac9ca58417ed8)

This seems like a useful macro. I think it could be nice to add to the libmultiprocess repo instead of the bitcoin repo, so the bitcoin cmake build would be simpler and the macro could also be used upstream to simplify:

https://github.com/chaincodelabs/libmultiprocess/blob/8da797c5f1644df1bffd84d10c1ae9836dc70d60/example/CMakeLists.txt#L5-L29
https://github.com/chaincodelabs/libmultiprocess/blob/8da797c5f1644df1bffd84d10c1ae9836dc70d60/test/CMakeLists.txt#L18-L28

Reply:
You read my mind :)